cmd/juju/controller/addmodel_test.go

```diff
@@ -209,6 +231,47 @@ func (s *AddModelSuite) TestCredentialsNoDefaultCloud(c *gc.C) {
     juju add-model \[flags\] \<model-name\> cloud\[/region\]`)
 }
 
+func (s *AddModelSuite) TestCredentialsOneCached(c *gc.C) {
```

Comment:
See `TestCredentialsOtherUserPassedThrough` and `TestCredentialsOtherUserPassedThroughWhenCloud`. Missing a test that shows that we *don't* upload when model owner != credential owner. Will add.